jooby/src/main/java/org/jooby/Jooby.java

```diff
@@ -3196,11 +3197,17 @@ static String logback(final Config conf) {
     if (conf.hasPath("logback.configurationFile")) {
       logback = conf.getString("logback.configurationFile");
     } else {
+      String env = conf.hasPath("application.env") ? conf.getString("application.env") : null;
+      if (env != null) {
+        URL classpathLogback = Thread.currentThread().getContextClassLoader().getResource("logback." + env + ".xml");
```

Comment:
Why `.getContextClassLoader`? We usually use: `getClass().getResource("/logback." + env + ".xml")`